### PR TITLE
eiji peido

### PR DESCRIPTION
**Pull request by pyumii**
ele eh nojento

### CODE REVIEW

**Comment by tayumii** [comment]
Mas que fedorrrrrrrrrrrrrrrrrrrrrrrrrrrrrr

**Comment by eijitkzw** [comment]
Quero ficar rico

File: patricia/novo1.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+eae
+isso eh mto legaaaalll
+#EIJI_peido+
```
Comment (fwfurtado) [inline patricia/novo1.md]: What do you think about adding a blank space between # and the text?

Comment (pyumii) [reply patricia/novo1.md]: blank space added

**Comment by massaoozono** [comment]
Vamos manter um bom nivel

**Pull request merged**